### DISCUSSION TITLE
Can't connect to WSA with ADB

### DISCUSSION THREAD

**Discussion by AlexanderSLoburev** [LSPosed/MagiskOnWSALocal#317] [Q&A] [answered]
Greetings, I installed the WSA according to your manual, but I can't connect to it:

`cannot connect to 127.0.0.1:58526: No connection could be made because the target machine actively refused it. (10061)`

I will be glad for any help!

**Comment by Howard20181** [comment, marked as answer]
1. Enable developor mode in WSA settings.

2. Reboot Windows.

**Reply by AlexanderSLoburev** [reply]
Thank you! It works)
